app/actions/websocket/category.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {deleteCategory, storeCategories} from '@actions/local/category';
+import {fetchCategories} from '@actions/remote/category';
+import DatabaseManager from '@database/manager';
+import {queryCategoriesById} from '@queries/servers/categories';
+
+type WebsocketMessage = {
+    broadcast: {
+        team_id: string;
+    };
+    data: {
+        team_id: string;
+        category?: string;
+        category_id: string;
+        updatedCategories?: string;
+        order?: string[];
+    };
+}
```

Comment:
I think this type name is already defined elsewhere so perhaps change the name to `WebsocketCategoriesMessage` or something